Resolve #11741, bbot update based on testing using BBOT v2.1.2 (#11742)
elastic/integrations: packages/bbot/_dev/build/docs/README.md

Once the BBOT scan is complete, the integration will ingest the results into Elastic.

This integration requires the external use of BBOT. You will have to download and run the tool apart from this integration. Once your scan is complete, this integration can ingest the results into Elastic using the BBOT `json` output module or the `http` output module aka. HTTP POST/webhooks.

> As of version `0.3.0` of this integration both BBOT `1.x` and `2.x` are supported. However all examples provided by the module documentation now assume use of BBOT `2.x`.

Reviewer: `0.3.0`?

Author: Draft version that was in use privately for awhile. Corrected to 1.1.2 now.

In order to support the conflicting BBOT NDJSON event formats the following occurs during ingest of BBOT `2.x` SCAN events:

> 1. The `bbot.data.scan` field, if received as a Map aka. object, is renamed to `bbot.data.scan_config`

Reviewer: ```suggestion
1. The `bbot.data.scan` field, if received as an object, is renamed to `bbot.data.scan_config`
```
